brave/brave-core
# Add method to increase contrast on Select

- Added method to decide which style do we have to show when someone opens **New Tab -> Customize (Clock Tab)**. 
- Modify brave-ui to match latest commit

Issue related on **brave-browser**: https://github.com/brave/brave-browser/issues/12061

![image](https://user-images.githubusercontent.com/13340320/110271876-9cb05580-7fa7-11eb-8f8f-537ba36ce184.png)



Resolves brave/brave-browser#12061

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Go to new tab
- Open Customize
- Go to Clock Tab in Dark Mode
- Check that text is clearly visible
